app/views/development/index.html.erb

```diff
@@ -1 +1,56 @@
-My page
+<%= content_for :title, "Dev page" %>
+
+<span class="govuk-caption-xl">Data analytics for</span>
+<h1 class="govuk-heading-xl">The page being analysed</h1>
+
+<%= render "govuk_publishing_components/components/govspeak" do %>
```

Comment:
I'm not sure we need govspeak here, unless we have a lot of content. If it's for styling the paragraph we can use `<p class="govuk-body">`

Reply:
The dev page isn't a final page, it's only included here for visibility of the component. I'll be using it as a testing ground as I build so this area will probably expand, so I'll not address this.